report/report.py

```diff
@@ -0,0 +1,101 @@
+from sisyphus import *
```

Comment:
Can you add an `__all__` entry s.t. `from report import *` does not in fact import everything.